azureVMCreateCommentVariables.md

```diff
@@ -0,0 +1,159 @@
+---
+title: 'Quickstart: Use the Azure CLI to create a Linux VM'
+description: In this quickstart, you learn how to use the Azure CLI to create a Linux virtual machine
+author: cynthn
+ms.service: virtual-machines
+ms.collection: linux
+ms.topic: quickstart
+ms.workload: infrastructure
+ms.date: 06/01/2022
+ms.author: cynthn
+ms.custom: mvc, seo-javascript-september2019, seo-javascript-october2019, seo-python-october2019, devx-track-azurecli, mode-api
+---
+
+<!-- 
```

Comment:
It seems that these should be hidden from in the main doc, is this a request of the docs team? I've always used them as an opportunity to describe what parameters are available to the user.